src/serverOperations/fetchFromBase.js

```diff
@@ -0,0 +1,30 @@
+import { SERVER_PATH } from '../constants';
+
+export async function fetchFromBase(path, params, method = 'get') {
```

Comment:
This doesn't look like fetch from "base" to me. It seems rather like `sendApiRequest` or `connectToApi` 🤔